kotlinx-coroutines-test/common/src/TestCoroutineScheduler.kt

```diff
@@ -177,6 +177,14 @@ public class TestCoroutineScheduler : AbstractCoroutineContextElement(TestCorout
         }
     }
 
+    /**
+     * Moves the virtual clock of this dispatcher forward by [the specified amount][delayTime], running the
+     * scheduled tasks in the meantime.
+     *
+     * @throws IllegalStateException if passed a negative [delay][delayTime].
+     */
+    public fun advanceTimeBy(delayTime: Duration): Unit = advanceTimeBy(delayTime.inWholeMicroseconds)
```

Comment:
I think this should use `delayTime.inWholeMilliseconds`, the parameter in the overload taking a `Long` is named `delayTimeMillis`.

Reply:
Yep, you're right. This PR is marked as a draft because it's undertested, so such silly mistakes are very likely.